src/ui/units/collections/components/CollectionContentTable/CollectionContentTable.tsx

```diff
@@ -61,6 +62,39 @@ type Props = {
     refreshContent: () => void;
 };
 
+const dropdownActions: {[key: string]: React.ReactNode} = {
+    move: (
+        <div className={b('dropdown-item')}>
+            <Icon data={ArrowRight} />
+            <div className={b('dropdown-text')}>{i18n('action_move')}</div>
```

Comment:
1) We shouldn't use `i18n` from non-react context, `i18n` might be uninitialized in this context. We need to move this into a react component and use `useMemo`.
2) Let's store only data, without tags copying:
```typescript
const dropdownActions = useMemo(() => {
    return {
        move: {
            title: i18n('action_move'),
            icon: ArrowRight,
        },
        ...
    }
}, []);
```

Reply:
yup, maybe better to create new component in other file and add props for setting - icon component and text?